src/stories/baseComponents/utils.tsx

```diff
@@ -0,0 +1,294 @@
+    name: 'second_lift',
+  },
+];
+
+const liftStateTemplate: RomiCore.LiftState = {
+  available_floors: ['L1', 'L2', 'L3', 'L4'],
+  available_modes: new Uint8Array(0),
+  current_floor: 'L1',
+  current_mode: RomiCore.LiftState.MODE_UNKNOWN,
+  destination_floor: 'L1',
+  door_state: RomiCore.LiftState.DOOR_CLOSED,
+  lift_name: 'main_lift',
+  lift_time: { sec: 0, nanosec: 0 },
+  motion_state: RomiCore.LiftState.MOTION_STOPPED,
+  session_id: '',
+};
+
+const liftStates: { [key: string]: RomiCore.LiftState } = {
+  main_lift: { ...liftStateTemplate },
+  second_lift: {
+    ...liftStateTemplate,
+    lift_name: 'second_lift',
+    motion_state: RomiCore.LiftState.MOTION_UP,
+  },
+};
+
+// End of lift utils
+
+/***
+ * Start of Robot utils
+ */
+const robotState: RomiCore.RobotState = {
+  battery_percent: 100,
+  location: {
+    level_name: 'L1',
+    t: { sec: 0, nanosec: 0 },
+    x: 8,
+    y: -4,
+    yaw: 0,
+  },
+  mode: { mode: RomiCore.RobotMode.MODE_IDLE },
+  model: '40_hours',
+  name: 'main_robot',
+  path: [],
+  task_id: 'taskA',
+};
+
+const robotStates: RomiCore.FleetState[] = [
+  {
+    name: 'main_fleet',
+    robots: [
+      { ...robotState },
+      {
+        ...robotState,
+        name: 'second_robot',
+        mode: { mode: RomiCore.RobotMode.MODE_CHARGING },
+      },
+      {
+        ...robotState,
+        name: 'third_robot',
+        mode: { mode: RomiCore.RobotMode.MODE_DOCKING },
+      },
+      {
+        ...robotState,
+        name: 'fourth_robot',
+        mode: { mode: RomiCore.RobotMode.MODE_EMERGENCY },
+      },
+      {
+        ...robotState,
+        name: 'fifth_robot',
+        mode: { mode: RomiCore.RobotMode.MODE_GOING_HOME },
+      },
+      {
+        ...robotState,
+        name: 'sixth_robot',
+        mode: { mode: RomiCore.RobotMode.MODE_MOVING },
+      },
+      {
+        ...robotState,
+        name: 'seventh_robot',
+        mode: { mode: RomiCore.RobotMode.MODE_PAUSED },
+      },
+      {
+        ...robotState,
+        name: 'eigth_robot',
+        mode: { mode: RomiCore.RobotMode.MODE_WAITING },
+      },
+    ],
+  },
+];
+
+// end of Robot utils
+
+export const componentDisplayStyle: StyleTyping = {
+  display: {
+    display: 'grid',
+    gridTemplateColumns: '1fr 3fr',
+  },
+  modeInfoPanel: {
+    padding: '2rem',
+  },
+  modeInfoItem: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    padding: '0.5rem',
+  },
+  modeInfoLink: {
+    marginTop: '0.5rem',
+    padding: '0.5rem',
+  },
+  aTag: {
+    textDecoration: 'none',
+    color: 'rgb(20, 116, 243)',
+  },
+};
+
+export const defaultStyles: StyleTyping = {
+  root: {
+    margin: '0 auto',
+    width: '40%',
+  },
+  heading: {
+    padding: '0.5rem',
+  },
+  aTag: {
+    textDecoration: 'none',
+    color: 'rgb(20, 116, 243)',
+  },
+};
+
+// colors used outside material ui
+export const colorPalette: StyleTyping = {
+  unknown: {
+    backgroundColor: '#cccccc',
```

Comment:
We need to have the actual components and the storybook components reference the same value, the actual components are still using hardcoded value, only the `design-decisions` story is using this.